benchalerts/benchalerts/conbench_dataclasses.py

```diff
@@ -62,32 +60,23 @@ def contender_benchmark_result_info(self) -> List[BenchmarkResultInfo]:
             assert self.compare_link
             return [
                 BenchmarkResultInfo(
-                    name=benchmark_result["contender"]["case_permutation"]
-                    if "analysis" in benchmark_result
-                    else benchmark_result["benchmark"],
```

Comment:
Thank you for cleaning up this mess

Reply:
I think it's a great idea to maintain backwards HTTP API compatibility in our clients, but in this case the HTTP API has diverged so much that I thought it was easier to just do the synchronized deployment this one time.